src/librustc/session/config.rs

```diff
@@ -841,6 +844,10 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                       `hir,typed` (HIR with types for each node).",
                      "TYPE"),
         opt::opt_u("", "show-span", "Show spans for compiler debugging", "expr|pat|ty"),
+        opt::include_u("", "include-arguments-from",
+                       "Read additional arguments from the given FILE or the \
+                        standard input if FILE is `-`. Arguments must be \
+                        separated by \\0.", "FILE", b'\0'),
```

Comment:
I've personally never seen anything other than `,`, `;`, `:`, whitespace (spaces and/or newlines) or `\0` used to delimit records. That said, if there's a dedicated character for it...